pylace/src/lib.rs

```diff
@@ -1179,6 +1179,71 @@ impl CoreEngine {
         Ok(())
     }
 
+    /// Remove Rows at the given indices.
+    ///
+    ///
+    /// Example
+    /// -------
+    ///
+    /// >>> import lace
+    /// >>> engine = lace.Engine('animals.rp')
+    /// >>> removed = engine.remove_rows(["wolf", "ox"])
+    /// >>> removed["index"]
+    ///
+    fn remove_rows(&mut self, rows: &PyList) -> PyResult<PyDataFrame> {
+        let remove: Vec<String> = rows
+            .into_iter()
+            .map(|x| x.str().map(|s| s.to_string()))
+            .collect::<PyResult<Vec<String>>>()?;
+
+        let row_idxs: Vec<usize> = remove
+            .iter()
+            .map(|row_name| {
+                self.engine.codebook.row_index(row_name).ok_or_else(|| {
+                    PyIndexError::new_err(format!(
+                        "{row_name} is not a valid row index"
+                    ))
+                })
+            })
+            .collect::<PyResult<Vec<_>>>()?;
+
+        let mut df = polars::frame::DataFrame::empty();
+        let index = polars::series::Series::new(
+            "index",
+            remove.iter().map(|x| x.clone()).collect::<Vec<String>>(),
```

Comment:
I can't verify this ATM since I don't have my Rust workspace here, but I have two questions:
1.) Can you not just use `remove.clone()` here?
2.) Do you even need to clone here? `remove` isn't used afterwards, right?